prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -137,6 +137,12 @@ class ActionHistory {
         messages.add(event)
     }
 
+    fun trackUsername(userName: String?) {
```

Comment:
Since I was adding the filename, I might as well add tracking of the okta authorized username, when its known (for example, when sending from the Upload page)